public/javascripts/SVValidate/src/panorama/Panorama.js

```diff
@@ -186,7 +187,14 @@ function Panorama (label) {
                 function (data, status) {
                     if (status === google.maps.StreetViewStatus.OK) {
                         document.getElementById("svv-panorama-date").innerText = moment(data.imageDate).format('MMM YYYY');
+                        // Make Terms of Use & Report a problem links on GSV clickable. Should only be done once.
+                        // https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2546
+                        if (!bottomLinksClickable) {
+                            $("#view-control-layer").append($('.gm-style-cc').slice(1, 3));
+                            bottomLinksClickable = true;
+                        } 
                     }
+                      
```

Comment:
Could you remove this new line? And could you actually just make it look like
```
} else {
```

which would fit the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide#general-guidelines).